functorch/functorch/csrc/DynamicLayer.h

```diff
@@ -18,12 +18,28 @@
 #include <functorch/csrc/ADInterpreters.h>
 #include <functorch/csrc/FunctionalizeInterpreter.h>
 
-// Forward declared bc I am lazy
+// Forward declared
 namespace c10 { struct AutogradMetaInterface; }
 
 namespace at {
 namespace functorch {
 
+// This file contains the implementation of functorch's interpreter stack.
+// See NOTE: [functorch interpreter stack] first before reading on.
```

Comment:
Is this note already in the code? I'm just seeing it referenced a lot of places without seeing it, so wanted to make sure it wasn't forgotten about

Reply:
Yes: https://github.com/pytorch/pytorch/blob/31ef8ddb8c4467f5b8698ef1eb9bb8bab7056855/functorch/functorch/csrc/Interpreter.h#L12